Sources/Box/Box.swift

```diff
@@ -0,0 +1,362 @@
+        let buttonProvider: ButtonConstructor?
+
+        /// Object responsible for providing colors in different environments and variants.
+        let colorProvider: ColorProvider
+
+        public static func == (lhs: Box, rhs: Box) -> Bool {
+            lhs.style == rhs.style && lhs.title == rhs.title
+        }
+
+        public func hash(into hasher: inout Hasher) {
+            hasher.combine(style)
+            hasher.combine(title)
+        }
+
+        public init(
+            style: Warp.BoxStyle,
+            title: String?,
+            subtitle: String,
+            link: ButtonConstructor? = nil,
+            button: ButtonConstructor? = nil,
+            colorProvider: ColorProvider = Config.colorProvider
+        ) {
+            self.style = style
+            self.title = title
+            self.subtitle = subtitle
+            self.linkProvider = link
+            buttonProvider = button
+            self.colorProvider = colorProvider
+        }
+
+        init(
+            style: Warp.BoxStyle,
+            title: String?,
+            subtitle: String,
+            linkProvider: ButtonConstructor?,
+            buttonProvider: ButtonConstructor?,
+            colorProvider: ColorProvider = Config.colorProvider
+        ) {
+            self.style = style
+            self.title = title
+            self.subtitle = subtitle
+            self.linkProvider = linkProvider
+            self.buttonProvider = buttonProvider
+            self.colorProvider = colorProvider
+        }
+
+        public var body: some View {
+            ZStack {
+                backgroundView
+
+                foregroundView
+                    .padding(.all, 24)
+            }
+            .cornerRadius(boxCornerRadius)
+            .accessibilityElement(children: .combine)
+            .modifier(
+                AccessibilityTraitBuilder(
+                    buttonProvider: buttonProvider,
+                    linkProvider: linkProvider
+                )
+            )
+            .modifier(AccessibilityButtonActionBuilder(buttonProvider: buttonProvider))
+        }
+
+        private var backgroundView: some View {
+            style.getBackgroundColor(from: colorProvider)
+                // View with rounded edges,
+                // and optional border.
+                .overlay(borderView)
+        }
+
+        @ViewBuilder
+        private var borderView: some View {
+            RoundedRectangle(cornerRadius: boxCornerRadius)
+                .stroke(style.getBorderColor(from: colorProvider), lineWidth: 2)
+        }
+
+        private var foregroundView: some View {
+            HStack(spacing: 8) {
+                toolTipIconView
+                    .accessibilityHidden(true)
+
+                informationView
+                    .padding(.leading, 16)
+
+                Spacer()
+            }
+        }
+
+        @ViewBuilder
+        private var toolTipIconView: some View {
+            if true {
```

Comment:
if there is no title this should be hidden too.

Reply:
Great catch, I will update it.
I will make it injectable.